docs/api/web-utils.md

```diff
@@ -16,11 +16,26 @@ Returns `string` - The file system path that this `File` object points to. In th
 
 This method superseded the previous augmentation to the `File` object with the `path` property.  An example is included below.
 
-```js
-// Before
-const oldPath = document.querySelector('input').files[0].path
+```js @ts-nocheck
+// Before (renderer)
+const file = document.querySelector('input[type=file]').files[0]
+alert(`Uploaded file path was: ${file.path}`)
+```
 
-// After
-const { webUtils } = require('electron')
-const newPath = webUtils.getPathForFile(document.querySelector('input').files[0])
+```js @ts-nocheck
+// After (renderer)
+const file = document.querySelector('input[type=file]').files[0]
+electron.showFilePath(file)
+
+// (preload)
+const { contextBridge, webUtils } = require('electron')
+
+contextBridge.exposeInMainWorld('electron', {
+  showFilePath (file) {
+    // It's best not to expose the full file path to the web content if
```

Comment:
I don't think we should document "you shouldn't do this" and then in the only example show them doing exactly that.

I.e. we should show people the Good Thing, not the Bad Thing and say figure out the good thing yourself

Reply:
@MarshallOfSound 
Thank you for your comment.
I agree with you. However, what is done in this example is actually a "good" thing (the full file path is not returned to the renderer process), so the document doesn't make readers figure out the good thing themselves.

I'm thinking about the two options:
- Remove the comment, because it is not necessary information for specific usage of the API, but rather general guidance concerning security and preload scripts.
- Keep the comment as is.
I prefer the former. Would that work for you?

Reply:
I'd keep the comment.

Reply:
Keeping the comment is also fine with me.

Reply:
I resolved a trivial conflict.
@MarshallOfSound @nikwen 
I'm wondering if I should make any changes, since it looks like you have differing opinions about the code comment.

Reply:
Personally, I'm fine with the example.

If Sam has concerns about it, I'm happy with changing it, too. In that case, a concrete suggestion for what to do different would be helpful. (I'd love to understand the concerns myself, too!)

Reply:
Thank you for thesuggestion. I will go with this approach:

> Just have a comment saying // do something with the path